wiki/Configuration:-Input.md

```diff
@@ -248,9 +258,13 @@ Settings specific to `touchpad`s:
 - `click-method`: can be `button-areas` or `clickfinger`, changes the [click method](https://wayland.freedesktop.org/libinput/doc/latest/clickpad-softbuttons.html).
 - `disabled-on-external-mouse`: do not send events while external pointer device is plugged in.
 
-Settings specific to `touchpad` and `mouse`:
+Settings specific to `touchpad`, `mouse`, `trackpoint`, and `trackball`:
 
 - `scroll-factor`: <sup>Since: 0.1.10</sup> scales the scrolling speed by this value.
+- `scroll-factor-horizontal`: <sup>Since: next release</sup> scales the horizontal scrolling speed by this value. If not set, falls back to `scroll-factor`. Negative values invert the scrolling direction.
```

Comment:
Question: is it more natural to have scroll-factor-horizontal and vertical *override* the main one, or *multiply* it? E.g. `scroll-factor 2 horizontal=-1` will currently result in horizontal=-1, but maybe it's more natural for it to result in -2? I'm on the fence myself, I think I'm leaning towards override.

Reply:
Yeah, I do think override is the most intuitive as well.